agent/runtime-attach/build.gradle.kts

```
@@ -20,5 +20,8 @@ tasks {
     from({
       agent.singleFile
     })
+    manifest {
+      attributes("Automatic-Module-Name" to "applicationinsights.runtime.attach")
```

I believe matching the package name is a good convention
```suggestion
      attributes("Automatic-Module-Name" to "com.microsoft.applicationinsights.attach")
```

---

I used the name that the automatic module naming used, for maximum compatibility.  If we change this, everyone (including us) who has previously set it up with Auto-naming will have to change their code.  I would prefer not to change this to something else as it would cause additional work for everyone using Modular Java who was able to get Automatic Modules to work with their project.

---

I tested this change, and while Gradle build still works and it still runs, IntelliJ reports that it can't find the module when I make the change above.  So while we could still use it if we match the package name, it's more convenient to use the one that Automatic Module Naming picks if it is working in that project so that we can compile in both IntelliJ and Gradle.  

If you still want to match the package name, we should add a note to the Changelog mentioning the difference. Let me know what you decide.  Thanks for the help on this.

---

thx @KrogerWalt! I'm afraid I don't have a good grasp on Java Modules, would you mind posting a Java Modules test project that uses runtime attach, that I can use to better understand the naming issue related to Intellij?

---

Renaming issue turned out to be a caching bug in the IntelliJ version I was using.  Updating to latest version fixed that.  This was the last hurdle,  so I will accept the change suggestion and add a line to the Changelog.  Thanks!